demosplan/DemosPlanCoreBundle/Logic/Statement/StatementDeleter.php

```diff
@@ -83,20 +84,27 @@ private function deleteOriginalStatement(Statement $originalStatement): void
         if (!$originalStatement->getChildren()->isEmpty()) {
             throw new InvalidArgumentException('Original-Statement to delete still has children.');
         }
-        $forReport = clone $originalStatement;
+        /** @var Connection $doctrineConnection */
+        $doctrineConnection = $this->getDoctrine()->getConnection();
+        $sql = 'DELETE FROM statement_import_email_original_statements WHERE original_statement_id = :id';
+        $prepareEmailToDelete = $doctrineConnection->prepare($sql);
+        $execute = $prepareEmailToDelete->executeQuery(['id' => $originalStatement->getId()]);
+        $execute->fetchAllAssociative();
 
-        $this->deleteStatementObject(
+        $deleteOriginal = $this->deleteStatementObject(
             $originalStatement,
             true,
             true
         );
-
-        $entry = $this->statementReportEntryFactory->createDeletionEntry($forReport);
-        $this->reportService->persistAndFlushReportEntries($entry);
-        $this->logger->info(
-            'generate report of deleteStatement(). ReportID: ',
-            ['identifier' => $entry->getIdentifier()]
-        );
+        $forReport = clone $originalStatement;
```

Comment:
do you intentionally clone the object after deleting?

Reply:
done in [cf306b7](https://github.com/demos-europe/demosplan-core/pull/3014/commits/cf306b7fdda03c47901525e7dccf6c7a3a461410)